orca-peering/src/main/kotlin/com/netflix/spinnaker/orca/peering/PeeringAgent.kt

```diff
@@ -189,6 +192,24 @@ class PeeringAgent(
     }
   }
 
+  /**
+   * If we have any custom peerers, invoke them
+   */
+  private fun invokeCustomPeerers() {
+    customPeerers.forEach { customPeerer ->
```

Comment:
it might make sense to define a `getPriority` method in `CustomPeerer` so that we can control the order in which they run, independently of the order Spring orders them beans

Reply:
I thought about it, and the ability to specify if you want to run before or after the rest of the peering runs. Ultimately, I decided it's added complexity for no reason.
If you have custom data that needs peering, you should do that in a single peerer. Reasoning about the order in which they run across multiple classes seems fraught.

I guess as I say this, I think the right approach here is to change List<CustomPeerer> to CustomPeerer to only allow a single one. Thoughts?

Reply:
If we did actually need that wouldn't using `@Order` on the bean itself do the trick?

Reply:
i'm changing this to only take a single custom peerer, it doesn't make sense to have multiple - if it does, we can change it then

Reply:
Either an explicit order or a single one are fine with me